### ISSUE TITLE
Reading specific rows from a large `sas7bdat` file

### ISSUE THREAD

**Issue by BERENZ** [junyuan-chen/ReadStatTables.jl#42]
Is there a way to add functionality to read specific rows from a large `sas7bdat` file? The issue I'm facing is that I have large SAS files (around 10GB) along with text files (an exact, flat copy of the SAS file). Based on the text file, I can specify the subset of rows that I'm interested in (around 10% of the file).

Another option is to specify a filter while reading, for example, reading rows based on a column. However, I understand that this may be more challenging to implement.

**Comment by BERENZ** [comment]
Hi, yes, but it would only work if the rows I want to select are in order. In my case, they're spread out over the dataset.

**Comment by junyuan-chen** [comment]
Yes. For reading the files, the iteration across rows is handled within the C library and there is no such an interface to skip rows depending on the values.
